### PR TITLE
DP-32118: Verify and fix conditional fields

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-****

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/modules/custom/mass_validation/mass_validation.module [hunk 1]
```diff
@@ -207,17 +211,24 @@ function mass_validation_form_node_form_alter(&$form, FormStateInterface $form_s
...
-      // we remove that error, but we keep other genuine errors connected to the form submission.
-      // The way we do this is inspired by code in the conditional_fields module.
+      // field_organizations is a multi value field,
+      // on which we get a false error if one or more
+      // values are present, but the first value is missing.
+      // If such a false error is present then
+      // we remove that error, but we keep other genuine errors
+      // connected to the form submission.
+      // The way we do this is inspired by code in
+      // the conditional_fields module.
```
Comment (weitzman) [inline docroot/modules/custom/mass_validation/mass_validation.module:221]: This change and several others are not desireable IMO. The 80 char limit is from old days of teletype. Editors can line wrap since about 1985. If your editor is reformatting unrelated code for line length, please disable that. Its OK for this PR.

File: composer.json [hunk 2]
```diff
@@ -450,8 +450,8 @@
             "drupal/inline_entity_form": {
                 "Add a setting to enable/disable inline editing of existing entities (https://www.drupal.org/project/inline_entity_form/issues/2913571#comment-12811088)": "https://www.drupal.org/files/issues/2018-10-11/ief_edit_existing_2913571_7.patch"
             },
-            "drupal/iframe_resizer": {
-                "Missing license URL (https://www.drupal.org/project/iframe_resizer/issues/3385378)": "https://www.drupal.org/files/issues/2023-09-05/iframe_resizer-missing_license_url-3385378-3.patch"
+            "drupal/paragraphs": {
+                "Nested paragraphs title gets cut off (https://www.drupal.org/project/paragraphs/issues/3246140)": "https://www.drupal.org/files/issues/2024-03-11/paragraphs-3246140-19-fix_paragraph_title_cutoff.patch"
             },
```
Comment (weitzman) [inline composer.json]: Please confirm that we intend to remove a patch and add a patch. Its hard for me to know. Also we intended to upgrade paragraphs module?

Comment (arthurbaghdas) [reply composer.json]: Yes, the iframe_resizer patch was duplicate in composer.json so we removed one of them. The paragraphs was updated and patched to solve the issue https://www.drupal.org/project/paragraphs/issues/3246140
